content/src/main/content/jcr_root/apps/acs-commons/components/authoring/component-cloner/clientlibs/touchui/.content.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--/*
+  ~  ACS AEM Commons Package
+  ~
+  ~ Copyright 2018 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+ */-->
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="cq.authoring.dialog"/>
```

Comment:
clientlibs in acs commons need to be opt-in only, so this will need to be a category specific to your feature.  Then in your documentation of the feature you will instruct users to create an empty clientlib that is in the category of `cq.authoring.dialog` and has your library as an `embed` or `dependency`

Reply:
I am not sure I am following. My understanding is that in Touch-UI, you must use the 'cq.authoring.dialog' category for it to 'register' and work properly?

Reply:
right, so to enable this feature a developer would create a simple client library with no code of its own, but in the category `cq.authoring.dialog` and with an `embed` of your library here.

Make sense?

Reply:
Got it, that way this doesn't get loaded by default. Instead forcing the user to manually 'activate' it.